### PR TITLE
Configure zsh

### PR DESCRIPTION
**Pull request by evgenTraytyak**

### CODE REVIEW

**Comment by evgenTraytyak** [comment]
#1 :up:

**Comment by evgenTraytyak** [comment]
#3 :up:

File: .zsh/aliases.zsh [hunk 1]
```diff
@@ -0,0 +1,82 @@
+
```
Comment (evgenTraytyak) [inline .zsh/aliases.zsh]: remove empty line :O

**Comment by dzhlobo** [comment]
@evgenTraytyak, thank you very much. I've made a lot of changes but your initial changes were very useful to start from.

❤️

**Comment by evgenTraytyak** [comment]
@Proghat no thanks at all 🍻

**Pull request merged**